elide-datastore/elide-datastore-aggregation/src/test/resources/configs/models/tables/SalesView.hjson

```diff
@@ -174,7 +174,7 @@
       [
         {
           name: region
-          to: RegionDetails
+          to: regionDetails
```

Comment:
What test is validating this change?

Reply:
https://github.com/yahoo/elide/blob/hjson_join_validation/elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/AggregationDataStoreIntegrationTest.java#L916
https://github.com/yahoo/elide/blob/hjson_join_validation/elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/integration/AggregationDataStoreIntegrationTest.java#L989
`customerRegionRegion` field is pulled from static model now. (https://github.com/yahoo/elide/blob/hjson_join_validation/elide-datastore/elide-datastore-aggregation/src/test/resources/configs/models/tables/SalesView.hjson#L57)